qcodes/instrument_drivers/Keysight/Keysight_N5230A.py

```diff
@@ -0,0 +1,160 @@
+from qcodes import VisaInstrument
+import numpy as np
+from qcodes.utils.validators import Numbers
+import time
+"""Driver for the Keysight PNA N5230A"""
+
+
+class N5230A(VisaInstrument):
+    """
+    This is the qcodes driver for Keysight/Agilent N5230A PNA-L Network Analyzer.
+    This should be simple to alter to work with other keysight PNAs
+    """
+
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name, address, terminator='\n', **kwargs)
+
+        self.add_parameter('get_trace',
+                           label='Get Trace',
+                           get_cmd=self._get_trace,
```

Comment:
it seems more clear to me (and a better design) to implement this as a method rather then as a parameter with only "get" command. What is the advantage of the current implementation apart from being quick-to-write for the developer?

Reply:
The current design (Loop etc. ) expect everything that returns data to be a parameter. But as it is here that is not really useful for that purpose unless it is expressed as an array or multi parameter. But the name of the parameter should probably not be get_trace but just trace if this is expressed as a parameter. It is odd to be able to do get_trace.get() imho